PeerioTechnologies/peerio-mobile
# fix: restore fileState.currentFile after file/folder move action is p…

…erformed [branch ch9255]

#### Relevant info and issue/PR links 
https://app.clubhouse.io/peerio/story/9255/mobile-files-tab-title-should-show-current-location-after-moving-a-file
#### Testing instructions  
Case 1:
Step 1: Create a folder A and create another nested folder B inside it. Tap on the action icon for folder B and select move.
Step 2: Move it to the root folder.
Result: The title of the file list (at the top) should show the correct title depending on your location in the files list, in this case, it should be the name of folder A.

Case 2:
1- Repeat step 1 in case 1.
2- Cancel the move action.
Result: Same as case 1.

----
### Repository owner


[PR guideline](https://github.com/PeerioTechnologies/peerio-icebear/blob/dev/docs/CONTRIBUTING.md)
